ci/gpu/build.sh

```diff
@@ -75,8 +75,8 @@ if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
     "dask/label/dev::dask" \
     "dask/label/dev::distributed"
 else
-  gpuci_logger "gpuci_mamba_retry install conda-forge::dask==${DASK_STABLE_VERSION} conda-forge::distributed==${DASK_STABLE_VERSION} conda-forge::dask-core==${DASK_STABLE_VERSION} --force-reinstall"
-  gpuci_mamba_retry install conda-forge::dask==${DASK_STABLE_VERSION} conda-forge::distributed==${DASK_STABLE_VERSION} conda-forge::dask-core==${DASK_STABLE_VERSION} --force-reinstall
+  gpuci_logger "gpuci_mamba_retry install conda-forge::dask>=${DASK_STABLE_VERSION} conda-forge::distributed>=${DASK_STABLE_VERSION} conda-forge::dask-core>=${DASK_STABLE_VERSION} --force-reinstall"
+  gpuci_mamba_retry install conda-forge::dask>=${DASK_STABLE_VERSION} conda-forge::distributed>=${DASK_STABLE_VERSION} conda-forge::dask-core>=${DASK_STABLE_VERSION} --force-reinstall
```

Comment:
> plus we want to keep it as it was for when we pin Dask again in the future.

Does this mean we will want to pin against only one version of `dask` for `dask-cuda` in the future too? I thought it was needed for `22.06` because of a buggy previous release i.e., `2022.05.1`

Reply:
I would likely say yes. It happened for 22.04, where we had `==2022.03.0`. Lately there have been so many breaking changes in every Dask release that has become impossible to support multiple versions. I had a longer comment on this in https://github.com/rapidsai/dask-cuda/issues/848#issuecomment-1147627964 , but I don't see us getting much confidence in multiple stable versions in the short-term given the fast pace Distributed has been changing. Maybe things will change and we will be able to re-evaluate this once we get close to release, but because we can't test multiple versions in CI that's probably risky anyway.

Reply:
Got it, This sounds good to me @pentschev. I will keep this in mind while I pin for future releases.